staging/kube-oidc-proxy/templates/service.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "kube-oidc-proxy.fullname" . }}
+  labels:
+{{ include "kube-oidc-proxy.labels" . | indent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: 443
```

Associated change that would go with the comment above.

```suggestion
      targetPort: https
```